content/en/container-orchestration.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concept
+---
+
+## What it is
+Container orchestration automates container deployment, management, scaling, and networking. It's primary goal is to manage the life cycle of containers and their dynamic environments. A conductor, like in an orchestra, directs a group of musicians who work collaboratively to perform a piece of music. Similarly containers in your application are orchestra musicians.
+
+## Problem it addresses 
+On behalf of the user, container orchestration organizes the compute, networking, and storage infrastructure. To ensure that applications are highly available and computing resources are used to their full potential, it does away with the requirement for direct manual orchestration in a cluster and automates the orchestration process.  
+
+## How it helps
+Container orchestration results in a more seamless customer experience by increasing productivity, resilience to failure, scalability, and flexibility. In addition to offering a platform for running, managing, and operating containers, it controls the container life cycle. For all container workloads and infrastructure, it offers a centralized management interface that also aids in logging and monitoring.
```

Comment:
This needs some work too. I (or someone else on our team) will provide more concrete feedback in the next few days.

Reply:
Great, looking forward to it.